app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,22 @@
+class PostsController < ApplicationController
```

Comment:
* [ ] As mentioned in the previous review: "Use a before_action in the posts controller to allow only authenticated users to create posts."